orienteer-graph/src/main/java/org/orienteer/graph/component/widget/GraphEdgesWidget.java

```diff
@@ -74,12 +81,14 @@ public void populateItem(Item<ICellPopulator<ODocument>> components, String s, I
 
         OrienteerDataTable<ODocument, String> table =
                 new OrienteerDataTable<ODocument, String>("edges", columns, vertexEdgesDataProvider, 20);
+        table.addCommand(new CreateEdgeCommand(new ResourceModel("command.create"),table, getModel()).setBootstrapType(BootstrapType.PRIMARY));
```

Comment:
LabelModel (I mean new ResourceModel("command.create")) is better to incorporate with in CreateEdgeCommand constructor. Please look how other end commands are organized.